### DISCUSSION TITLE
Issue with deployed rke2 second Ingress-Nginx Controller inside a Kubernetes cluster running on bare-metal.

### DISCUSSION THREAD

**Discussion by ghost** [nginx/kubernetes-ingress#6836] [Q&A]
We have deployed two Nginx ingress controller in our cluster. The first ingress controller works fine but we are facing issue with second one. When accessing the service through second ingress controller (http://x3.abc.com/sample), we get This site can’t be reached x3.abc.com took too long to respond. error. We followed the same step as the first ingress controller including assigning different IP address using metallb and different ingress class in different namespace.

Details:
RKE2 Version: v1.28.11
Kubernetes Version: Client Version: v1.30.3 Server Version: v1.28.11
NGINX Ingress controller
  Release:       v1.10.1-hardened1
  Build:         git-b48c66a2f
  Repository:    https://github.com/rancher/ingress-nginx
  nginx version: nginx/1.25.3

Metallb Version: v0.14.5

For testing purpose we deployed a simple nginx image using below yaml files. Everything is up and running, No error logs in pods
, service mapped correct endpoints and ingress resource got external IP assigned by Second nginx ingress controller But when we try to access http://x3.abc.com/sample gives Site can't be reached error.
Deployment.yaml
![image](https://github.com/user-attachments/assets/6f04a397-17ed-4039-b6f8-f87361cb1c77)

Service.yaml
![image](https://github.com/user-attachments/assets/b4d098e1-a96b-4fd7-89df-48cdb96d9d2c)

Ingress.yaml
![image](https://github.com/user-attachments/assets/10f5a481-6fdd-49f8-9201-beeef7e90c34)

When we change ingress class and host of the ingress resource with First ingress controller rest everything same then service is accessible via http://y3.abc.com/sample gives Welcome to Nginx

Ingress2.yaml
![image](https://github.com/user-attachments/assets/5640e860-f6be-4227-a679-4e68e3bfbb26)
we would appreciate it if someone could help us to find out root cause of this issue. I am ready to provide any additional details if required. Thanks in advance!!

**Comment by j1m-ryan** [comment]
Hi @Ganesh-hub-5, I believe you are referring to the [community ingress controller](https://github.com/kubernetes/ingress-nginx) based on your annotations and ingress controller version. This is a separate project to this so you can make an issue there for it.
